src/gtk/webview_webkit2.cpp

```diff
@@ -1093,13 +1099,81 @@ wxString wxWebViewWebKit::GetPageText() const
     return wxString();
 }
 
-void wxWebViewWebKit::RunScript(const wxString& javascript)
+static void wxgtk_run_javascript_cb(WebKitWebView *,
+                                    GAsyncResult *res,
+                                    GAsyncResult **res_out)
 {
+    g_object_ref(res);
+    *res_out = res;
+}
```

Comment:
Sorry for the delayed comment...
The function signature of wxgtk_run_javascript_cb() should probably be updated to match what was done here:
https://github.com/wxWidgets/wxWidgets/commit/59acf5ac27b9d95dbe95b7e86695a3a81bbd2a7b#diff-5f0febfdd2bcba8545ac117908300135

See also:
https://developer.gnome.org/gio/stable/GAsyncResult.html#GAsyncReadyCallback